### PR TITLE
docs: Move Automated Self Checkout use case to documentation repo

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
- [x] No Employer Confidential details are added

## What requirement is this design document for?
Move Automated Self Checkout use case to documentation repo

#**issue_number**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -45,6 +45,12 @@ nav:
   - Performance Tools:
       - Benchmarking: 'performance-tools/benchmark.md'
       - Benchmarking APIs: 'performance-tools/api-docs.md'
+  - Use Cases: 
+      - Automated Self Checkout: 'use-cases/automated-self-checkout/automated-self-checkout.md'
+      - Getting Started: 'use-cases/automated-self-checkout/getting_started.md'
+      - Advanced Settings: 'use-cases/automated-self-checkout/advanced.md'
+      - Performance: 'use-cases/automated-self-checkout/performance.md'
```
Comment (jim-wang-intel) [inline mkdocs.yml:52]: should these navigation items become sub-items of the first one: automated self-checkout?

File: docs_src/use-cases/automated-self-checkout/advanced.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Advanced Settings
+
+## Applying Environment Variables to Run Pipeline
```
Comment (NeethuES-intel) [inline docs_src/use-cases/automated-self-checkout/advanced.md:3]:
```suggestion
## Applying Environment Variables(EV) to Run Pipeline
```

File: docs_src/use-cases/automated-self-checkout/advanced.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+EV can be applied in two ways:
+
+    1. As a Docker Compose environment parameter input 
+    2. In the env files
+
+The input parameter will override the one in the env files if both are used.
+
+### Run with custom environment variables
```
Comment (NeethuES-intel) [inline docs_src/use-cases/automated-self-checkout/advanced.md:12]:
```suggestion
### Run with Custom Environment Variables
```

File: mkdocs.yml (hunk 1 above)
Comment (brian-intel) [reply mkdocs.yml:52]: Good idea. That will allow us to add additional use cases in the future.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**